core/lib/resources/presence/presence_manager.js

```diff
@@ -216,13 +201,11 @@ PresenceManager.prototype._implicitDisconnect = function(socketId, userId,
 
 PresenceManager.prototype.isOnline = function(message) {
   // Return with online, or with sentry, or with unexpired legacy msg
-  return (message.online && (message.sentry ||
-       (message.at && message.at >= (Date.now() - PresenceManager.autoPubTimeout))));
+  return (message.online && message.sentry);
 };
 
 PresenceManager.prototype.isExpired = function(message) {
-  return (message.online && !message.sentry &&
-      message.at && message.at < (Date.now() - PresenceManager.autoPubTimeout));
+  return (message.online && !message.sentry);
```

Comment:
@vanchi-zendesk The link doesn't take me to the specific section. Can you write the name of the section so I can find it?